### DISCUSSION TITLE
[Feature] Option to hide the "Memories" section

### DISCUSSION THREAD

**Discussion by Arche151** [immich-app/immich#3622] [Feature Request]
### The feature

I am kindly asking for the addition of the option to hide the Memories section.

For me, Memories have been super distracting and weird, because they show random pictures from years ago, that I don't want to see at the top of my gallery. 

<img width="1273" alt="Screenshot 2023-08-09 at 14 59 32" src="https://github.com/immich-app/immich/assets/124050126/e6489bb9-b1b2-4732-97eb-0c72d6c3250c">

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by etnoy** [comment]
I concur. In many cases the memories feature can be distracting. Perhaps there could be a cutoff so that photos older than X years are not shown as memories?

**Reply by bo0tzz** [reply]
I don't think a cutoff is appropriate, just because a photo is old doesn't mean you don't want to see it ;) I think just a toggle in the user settings to hide memories entirely should do the task.

**Comment by jrasm91** [comment]
Duplicate of #3187
